### ISSUE TITLE
500 internal error

### ISSUE THREAD

**Issue by TeckniX** [russellquinn/casein#57]
Rails 4.2.1
Ruby 2.2.0p0
Mac OSX 10.10.3
casein ~>5.1.1

I get the following browser error, when going to http://localhost:3000/casein
```html
500 Internal Server Error
If you are the administrator of this website, then please read this web application's log file and/or the web server's log file to find out what went wrong.
```

With the following logs within the server logs
```bash
=> Rails 4.2.1 application starting in development on http://localhost:3000
=> Run `rails server -h` for more startup options
=> Ctrl-C to shutdown server
[2015-06-18 15:30:31] INFO  WEBrick 1.3.1
[2015-06-18 15:30:31] INFO  ruby 2.2.0 (2014-12-25) [x86_64-darwin14]
[2015-06-18 15:30:31] INFO  WEBrick::HTTPServer#start: pid=16170 port=3000


Started GET "/casein" for ::1 at 2015-06-18 15:30:35 -0400
  ActiveRecord::SchemaMigration Load (0.1ms)  SELECT "schema_migrations".* FROM "schema_migrations"

ArgumentError (wrong number of arguments (2 for 1)):
  actionpack (4.2.1) lib/action_dispatch/http/request.rb:338:in `parse_query'
  rack (1.6.3) lib/rack/request.rb:191:in `GET'

```

Any idea what needs to be changed/updated?

**Comment by TeckniX** [comment]
Closing as it may be a setup issue and not specific to casein

**Comment by TeckniX** [comment]
For legacy purposes, the fix was to get rack back down to 1.6.2:
Add to your gemfile `gem 'rack', '1.6.2'`
Run: `bundle update rack` from the command-line
